api-guidelines/async/format/test-extension/rules/may-use-test-extension.md

```diff
@@ -17,13 +17,15 @@ This extension defines the `test` context attribute that event producers can pro
 - Type: `String`
 - Description: A string in the format `<test-scope>.<test-type>`. Both `test-type` and `test-scope` must not contain a `.` (dot).
   - `test-scope`: The scope of the executed test. Possible values include but are not limited to the name of a single context or a group of contexts (e.g., `orderprocessing`).
-  - `test-type`: The type of the test. Currently, the following types are defined; additional types may be added in the future:
+  - `test-type`: The type of the test. Currently, the following common types are defined:
     - `PerformanceTest` Automatically created test events with large datasets.
     - `SystemTest` Automatically created test events to test specific business cases within one team.
     - `End2EndTest` Automatically created test events to test specific business cases within multiple teams.
     - `ManualTest` Manually created test events.
+    
+    Additional test types may be used. They must be listed in the event documentation of the API provider
```

Comment:
If a new custom `test-type` is defined by a specific API Provider, do clients that merely forward the test event (according to https://api.otto.de/portal/guidelines/r200022) also need to document this new custom `test-type` in their event documentation?

Reply:
Good point...

Reply:
In my point of view the test header should be forwarded as is, even custom ones. If no one opposes, I can add it to the text

Reply:
We talked about it in the API COP and agreement is that the change can be added. I checked the rule for the forwarding and in my opinion it is already covered and no change is necessary ( - [MUST forward `test` context attribute](./must-forward-test-context-attribute.md) )